ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -2437,3 +2437,36 @@ def test_numpy_instance_abs__(
         frontend=frontend,
         frontend_method_data=frontend_method_data,
     )
+
+ 
+# __len__
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        min_num_dims=1,
+        max_num_dims=5,
+    ),
+)
+def test_numpy_instance_len__(
+    dtype_and_x,
+    as_variable,
+    native_array
```

Comment:
Inlude more flags like num_positional_args_method, frontend_method_data, and frontend.